docs/AUTHORS.md

```diff
@@ -37,7 +37,6 @@ A note on "decision making authority". This is related to submitting PRs and the
 |Name           |GiHub handle   |Decision making                          |Badges of honor (tag for questions) |
 |:--------------|:--------------|:----------------------------------------|------------------------------------|
 |Jamie Davis    |@davisjam      |Seeker of Security                       |                                    |
-|Steven         |@styfle        |Open source, of course and GitHub Guru   |                                    |
```

Comment:
Why did you remove me? Am I not allowed to review PRs?

Reply:
Trust me, if it were that big of a shift...it would have been a separate PR. ;)